File: WebRoot/com/tenze/pms/facade/dto/bi/BiMerchantNoDto.js
_fillInDataModel(
        'com.tenze.pms.facade.dto.bi.BiMerchantNoDto',
        {
            table:Ext_UiTable({
                beanId:'biMerchantNoId',
                beanName:'biMerchantNoName',
            })
            ,
            fields:[
                Ext_UiField({type:'string',name:'id',text:'id',xtype:'hidden',hidden:true,allowBlank:true,queryFlag:true}),
                Ext_UiField({type:'string',name:'version',text:'version',xtype:'hidden',hidden:true,allowBlank:true,queryFlag:true}),
                Ext_UiField({type:'string',name:'createtime',text:'createtime',xtype:'hidden',hidden:true,allowBlank:true,queryFlag:true}),
                Ext_UiField({type:'string',name:'updatetime',text:'updatetime',xtype:'hidden',hidden:true,allowBlank:true,queryFlag:true}),
                Ext_UiField({type:'string',name:'fdelFlag',text:'fdelFlag',xtype:'hidden',hidden:true,allowBlank:true,queryFlag:true}),
                 Ext_UiField({type:'string',name:'merId',text:'merId',xtype:'hidden',hidden:true,allowBlank:true,queryFlag:true}),
                 Ext_UiField({type:'string',name:'merName',text:'商家名称',xtype:'btnfield',width:200,hidden:false,allowBlank:false,queryFlag:true}),
                 Ext_UiField({type:'string',name:'noName',text:'商户编号',xtype:'textfield',width:200,hidden:false,allowBlank:false,queryFlag:true}),
                 Ext_UiField({type:'string',name:'merTenantName',text:'商户名称',xtype:'textfield',width:200,hidden:false,allowBlank:false,queryFlag:true}),
            ]
    }        
);

File: bi/com/tenze/pms/facade/dto/bi/BiMerchantNoDto.java
package com.tenze.pms.facade.dto.bi;


import java.util.Date;
import com.tenze.background.common.entity.BaseEntity;

/**
 * TODO
 * @author TenzeTseng
 *
 */
public class BiMerchantNoDto extends BaseEntity{

    private Long id;
    private Integer version;
    private Date createtime;
    private Date updatetime;
    private Integer fdelFlag;
    private Long merId;
    private String merName;
    private String noName;
    private String merTenantName;

public BiMerchantNoDto(){}

public BiMerchantNoDto(Long id){this.id=id;}

public BiMerchantNoDto(Long id, Integer version, Date createtime, Date updatetime, Integer fdelFlag, Long merId, String merName, String noName){
         this.id = id;
         this.version = version;
         this.createtime = createtime;
         this.updatetime = updatetime;
         this.fdelFlag = fdelFlag;
         this.merId = merId;
         this.merName = merName;
         this.noName = noName;
}

    public void setId(Long id){
        this.id = id;
    }

    public Long getId(){
        return this.id;
    }

    public void setVersion(Integer version){
        this.version = version;
    }

    public Integer getVersion(){
        return this.version;
    }

    public void setCreatetime(Date createtime){
        this.createtime = createtime;
    }

    public Date getCreatetime(){
        return this.createtime;
    }

    public void setUpdatetime(Date updatetime){
        this.updatetime = updatetime;
    }

    public Date getUpdatetime(){
        return this.updatetime;
    }

    public void setFdelFlag(Integer fdelFlag){
        this.fdelFlag = fdelFlag;
    }

    public Integer getFdelFlag(){
        return this.fdelFlag;
    }

    public void setMerId(Long merId){
        this.merId = merId;
    }

    public Long getMerId(){
        return this.merId;
    }

    public void setMerName(String merName){
        this.merName = merName;
    }

    public String getMerName(){
        return this.merName;
    }

    public void setNoName(String noName){
        this.noName = noName;
    }

    public String getNoName(){
        return this.noName;
    }

	public String getMerTenantName() {
		return merTenantName;
	}

	public void setMerTenantName(String merTenantName) {
		this.merTenantName = merTenantName;
	}

    
}